source/download/alternate_downloads.md

```diff
@@ -107,10 +107,10 @@ For a standalone engine this means basically:
 You must now install at least one Node or Host to act as hypervisors. Enterprise features like migration and high availability
 require more than one Host.
 
-oVirt supports two types of [Hosts](/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_cockpit_web_interface/#Installing_Hosts_for_RHV_SHE_cockpit_deploy):
+oVirt supports two types of [Hosts](/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/#Installing_Red_Hat_Virtualization_Hosts_SHE_cli_deploy):
```

Comment:
Seems that this is not the correct section link since it points to section `6.1.1 Installing oVirt Nodes` instead of `6. Installing Hosts for oVirt` as was for deprecated cockpit guide:

```suggestion
.....
/#Installing_Hosts_for_RHV_SHE_cli_deploy
```